Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "Clibgit",
+        "repositoryURL": "git@github.com:stonehouse/Clibgit.git",
```

Comment:
Let's include this in the SwiftGit2 repository instead of using an external dependency.

Reply:
So, with Swift Package Manager it needs a separate repo to define the dependency. It's kind of weird now, rather than system modules being just another product, it is at the root of the package. I'm happy for you to fork my repo and then the dependency is owned by the same account. But as far as I can tell, you need a separate git repo to wrap the C binary. It's actually discussed in the section of the docs that I linked below.